ravenml/data/helpers.py

```diff
@@ -0,0 +1,274 @@
+import os
+import shutil
+import pandas as pd
+import json
+from pathlib import Path
+from colorama import Fore
+from ravenml.utils.question import cli_spinner, user_selects, user_confirms, user_input
+from ravenml.utils.io_utils import copy_data_locally, download_data_from_s3
+from ravenml.utils.config import get_config
+
+def default_filter_and_load(**kwargs):
+    bucketConfig = get_config()
+    image_bucket_name = bucketConfig.get('image_bucket_name')
+
+    cli_spinner("Loading metadata...", ingest_metadata, kwargs['imageset'], image_bucket_name, kwargs['cache'])
+
+    tags_df = load_metadata(kwargs["metadata_prefix"], kwargs['cache'])
+    filter_metadata = {"groups": []}
+    
+    if kwargs['filter']:
```

Comment:
I will probably move this to a new function that returns the filter groups do increase readability